config.map.json

```
@@ -28,6 +28,7 @@
     "*/CodeQwen1.5-*-Chat-GGUF/*.gguf": "chatml.preset.json",
     "*/Hermes-2-Pro-Llama-3-*/*.gguf": "chatml.preset.json",
     "*/falcon*-GGUF/*.gguf": "falcon.preset.json",
-    "*/granite-*-GGUF/*.gguf": "granite.preset.json"
+    "*/granite-*-GGUF/*.gguf": "ibm_granite.preset.json",
+    "*/starcoder2-*-GGUF/*.gguf": "starcoder_2_instruct.preset.json"

```

```suggestion
    "*/starcoder2-*-instruct-*/*.gguf": "starcoder_2_instruct.preset.json"
```

Example models:
- https://huggingface.co/lmstudio-community/starcoder2-15b-instruct-v0.1-GGUF
- https://huggingface.co/bartowski/starcoder2-15b-instruct-v0.1-GGUF
- https://huggingface.co/Kquant03/TechxGenus-starcoder2-15b-instruct-GGUF